scripts/globals/combat/treasure_hunter.lua

```diff
@@ -0,0 +1,67 @@
+xi = xi or {}
+xi.combat = xi.combat or {}
+xi.combat.treasureHunter = xi.combat.treasureHunter or {}
+
+-- https://forum.square-enix.com/ffxi/threads/56550
+local treasureHunterTable =
```

Comment:
Make these local tables attached to the global so they can be more easily modified by modules 👍

Reply:
Done